.travis.yml

```diff
@@ -58,8 +59,10 @@ install:
   - ls -l file_metadata/datafiles
 
 script:
-  - flake8 setup.py setupdeps.py file_metadata tests
-  - python -m pytest --cov ;
+  - flake8 --verbose --show-source --statistics --benchmark --max-complexity 10 setup.py setupdeps.py file_metadata tests
+  - radon mi setup.py setupdeps.py file_metadata tests
```

Comment:
pytest only reports complexity issues with code it runs and reports it as failures while flake8 checks the whole code processed. So I prefer the flake8 check but reporting of pytest is nicer.

Radon in this case calculates the `Maintainability Index = MAX(0,(171 – 5.2 * log(Halstead Volume) – 0.23 * (Cyclomatic Complexity) – 16.2 * log(Lines of Code))*100 / 171)`. As you can see from [1] this metric considers the Halstead Volume and Lines of Code as well. So I would keep it to have a first fast view.

[1] https://blogs.msdn.microsoft.com/zainnab/2011/05/26/code-metrics-maintainability-index/